packages/shared-ux/prompt/no_data_views/impl/src/no_data_views.component.tsx

```diff
@@ -7,95 +7,205 @@
-      />
-    </p>
+  const icon = <EsqlIllustration />;
+
+  const title = (
+    <FormattedMessage
+      id="sharedUXPackages.noDataViewsPrompt.esqlPanel.title"
+      defaultMessage="Query your data with ES|QL"
+    />
   );
 
-  const footer = dataViewsDocLink ? <DocumentationLink href={dataViewsDocLink} /> : undefined;
+  const description = (
+    <FormattedMessage
+      id="sharedUXPackages.noDataViewsPrompt.esqlExplanation"
+      defaultMessage="ES|QL is a next-generation transformative piped query language and engine developed by Elastic. ES|QL simplifies workflows and advanced searches while accelerating query response for efficient data processing."
+    />
+  );
 
-  // Load this illustration lazily
-  const Illustration = withSuspense(
-    React.lazy(() =>
-      import('./data_view_illustration').then(({ DataViewIllustration }) => {
-        return { default: DataViewIllustration };
-      })
-    ),
-    <EuiPanel color="subdued" style={{ width: 226, height: 206 }} />
+  const footer = (
+    <>
+      <EuiButton onClick={onTryESQL} fill={true} data-test-subj="tryESQLLink">
+        <FormattedMessage
+          id="sharedUXPackages.noDataViewsPrompt.tryEsqlText"
+          defaultMessage="Try ES|QL"
+        />
+      </EuiButton>
+      <EuiHorizontalRule />
+      {esqlDocLink && <DocumentationLink href={esqlDocLink} data-test-subj="docLinkEsql" />}
+    </>
   );
 
-  const icon = <Illustration />;
-  const actions = (
-    <NoDataButtonLink {...{ onClickCreate, canCreateNewDataView, onTryESQL, esqlDocLink }} />
+  return (
+    <EuiCard
+      hasBorder={true}
+      data-test-subj="noDataViewsPromptTryEsql"
+      display={emptyPromptColor}
+      {...{ icon, title, description, footer }}
+    />
   );
+};
 
+/**
+ * A presentational component that is shown in cases when there are no data views created yet.
+ */
+export const NoDataViewsPrompt = ({
+  onClickCreate,
+  canCreateNewDataView,
+  dataViewsDocLink,
+  onTryESQL,
+  esqlDocLink,
+  emptyPromptColor = 'plain',
+}: NoDataViewsPromptComponentProps) => {
   return (
-    <EuiEmptyPrompt
+    <EuiFlexGroup
+      alignItems="center"
+      justifyContent="spaceEvenly"
       data-test-subj="noDataViewsPrompt"
-      layout="horizontal"
-      css={css`
-        max-width: ${MAX_WIDTH}px !important; // Necessary to override EuiEmptyPrompt to fit content
-        flex-grow: 0;
-      `}
-      color={emptyPromptColor}
-      {...{ actions, icon, title, body, footer }}
-    />
+    >
+      <EuiFlexItem
+        css={css`
+          max-width: ${MAX_WIDTH}px;
+        `}
+      >
+        <EuiText>
+          <EuiTextAlign textAlign="center">
+            <h2>
+              <FormattedMessage
+                id="sharedUXPackages.noDataViewsPrompt.youHaveData"
+                defaultMessage="You have data in Elasticsearch."
+              />
+              <EuiSpacer size="s" />
+              <EuiText size="s">
```

Comment:
Increase the size of this text a little

```suggestion
              <EuiText size="m">
```